librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -394,6 +394,68 @@ commands:
           - RZ_OUTPUT_MODE_RIZIN
           - RZ_OUTPUT_MODE_JSON
         args: []
+      - name: avg
+        summary: Global variables
+        subcommands:
+          - name: avg
+            summary: show global variables
+            type: RZ_CMD_DESC_TYPE_ARGV_STATE
+            cname: analysis_print_global_variable
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
+              - RZ_OUTPUT_MODE_JSON
+            args:
+              - name: var_name
+                type: RZ_CMD_ARG_TYPE_STRING
```

Comment:
let's add right now a new type in RzCmdArgType (e.g. `RZ_CMD_ARG_TYPE_GLOB_VAR`) and use it wherever a global variable name is used as argument. So when we want to implement autocompletion it will be easy to do. Also, it makes it clearer from the beginning that these are not just regular strings.

Reply:
This has been implemented now in #1577.